<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RogaSkillEditor</class>
 <widget class="QMainWindow" name="RogaSkillEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RogaSkillEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Desc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Skill Animation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Casting Animation</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Aim Target</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aim Grid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Chain</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Drain</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Multi-Target Animation</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Delay</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_3"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="AOECheckBox">
            <property name="text">
             <string>Area of Effect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="AOESpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Damage List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Min</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Max</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Random</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Hit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Critical</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Attr</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>DamageDef</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox_5">
        <property name="title">
         <string>Targets</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Group type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonOne">
             <property name="text">
              <string>one</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonTwo">
             <property name="text">
              <string>two</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <property name="usesScrollButtons">
            <bool>false</bool>
           </property>
           <property name="tabsClosable">
            <bool>false</bool>
           </property>
           <property name="movable">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="tabWidgetOne">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <attribute name="title">
             <string>One</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>12</number>
             </property>
             <property name="rightMargin">
              <number>12</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Selection Type</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboBox_4">
                 <item>
                  <property name="text">
                   <string>Single</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Side</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>All</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Field</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="gridGroupBox_4">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="title">
                <string>Target Type</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBox_2">
                  <property name="text">
                   <string>Self</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="checkBox_3">
                  <property name="text">
                   <string>Ally</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QCheckBox" name="checkBox_4">
                  <property name="text">
                   <string>Opponent</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="checkBox_6">
                  <property name="text">
                   <string>Alive</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="checkBox_5">
                  <property name="text">
                   <string>Dead</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabWidgetTwo">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <attribute name="title">
             <string>Two</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="leftMargin">
              <number>12</number>
             </property>
             <property name="rightMargin">
              <number>12</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Selection Type</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboBox_5">
                 <item>
                  <property name="text">
                   <string>Single</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Side</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>All</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Field</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="gridGroupBox_2">
               <property name="title">
                <string>Target Type</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_14">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBox_37">
                  <property name="text">
                   <string>Self</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="checkBox_38">
                  <property name="text">
                   <string>Alive</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QCheckBox" name="checkBox_39">
                  <property name="text">
                   <string>Opponent</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="checkBox_40">
                  <property name="text">
                   <string>Ally</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="checkBox_41">
                  <property name="text">
                   <string>Dead</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AOECheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>AOESpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
